assets/policy-extras/shaping.toml

```
@@ -122,7 +122,7 @@ match=[{MXSuffix=".yahoodns.net"}]
 max_deliveries_per_connection = 20
 
 [[provider."yahoo".automation]]
-regex = "\\[TS04\\]"
+regex = "\\[TS?(S)04\\]"
```

FWIW, I think you can simplify this to:

```suggestion
regex = "\\[TS?S04\\]"
```

---

Amended as suggested.